IPython/core/pylabtools.py

```diff
@@ -346,6 +377,10 @@ def find_gui_and_backend(gui=None, gui_select=None):
             gui = gui_select
             backend = backends_[gui]
 
+    # Since IPython 8.23.0 use None for no gui event loop rather than "inline".
+    if gui == "inline":
+        gui = None
+
```

Comment:
Should we emit a warning here, to make sure that things like backend_to_gui never return `"inline"` or similar ?

Reply:
We only get to this code if we are using "old" Matplotlib (i.e. before the `backend_registry` changes), in which case we will have `gui == "inline"` for the inline backend. There will already have been a `DeprecationWarning` issued for accessing `backend2gui` (a few lines above) and a second warning here might be more confusing than helpful.

Reply:
I will add an extra comment in the code here though.